pages/api/donation-backup.js

```diff
@@ -0,0 +1,30 @@
+const handler = (req, res) => {
+	const { body, method } = req;
+	try {
+		if (method === 'POST') {
+			fetch(process.env.MONGO_DONATION_URL, {
+				method: 'POST',
+				headers: {
+					'Content-Type': 'application/json',
+					'Access-Control-Request-Headers': '*',
+					'api-key': process.env.MONGO_DONATION_API_KEY,
+				},
+				body: JSON.stringify({
+					collection: process.env.MONGO_DONATION_COLLECTION,
+					database: process.env.MONGO_DONATION_DATABASE,
+					dataSource: process.env.MONGO_DONATION_DATA_SOURCE,
+					document: body,
+				}),
+			})
+				.then(response => response.json())
+				.then(data => console.log(data))
+				.catch(error => console.error('Error:', error));
```

Comment:
Is it possible to add sentry here to capture error?

Reply:
Having issues with both servers simultaneously is uncommon, and in this scenario, there's little we can do. The MongoDB server, being highly stable and hosted by AWS, limits our intervention options. While I will make an addition, its effectiveness might be limited.